### ISSUE TITLE
About parameters setting for ablation experiments

### ISSUE THREAD

**Issue by babble2** [retsuh-bqw/SRFormer-Text-Det#2]
![1](https://github.com/retsuh-bqw/SRFormer-Text-Det/assets/122713337/1622127a-021d-4c97-a50d-e99cfb673a29)
Excuse me，when performing ablation experiments in table 2 of the paper, are the parameters set correctly?

**Comment by retsuh-bqw** [comment]
Please refer to configs/SRFormer/TotalText/R_50_poly.yaml, and use (totaltext_poly_train_rotate_pos) as the training data when performing Tab.2. 
In addition, if you have limited computing resources and can only use BatchSize=1, you need to consider rescaling the learning rate, to avoid the gradient stochasticity impact on the optimization (but there is no guarantee that you can reproduce the results under bs=8).
